src/Umbraco.Web.UI.Client/src/common/services/blockeditormodelobject.service.js

```diff
@@ -532,6 +542,7 @@
 
                     delete this.layout;
                     delete this.data;
+                    delete this.settingsData;
```

Comment:
Is this correct it doesn't seem right? I'm trying to debug some issues with these changes since it's not submitting data properly. We have this.data.settingsData and this.data.contentData, yet this is deleting this.settingsData and this.content but I don't even thing this.content exists? hrm, there's also a few other references to this.content. I'll keep debugging but i think some things in this code are missing from moving to the new structure  correctly.